src/core/pd_repo.js

```diff
@@ -0,0 +1,83 @@
+import mongoose from "mongoose";
+
+import data_chunks from "../constants/pd_constants.js";
+
+import logger from "./pd_logger.js";
+
+class repo {
+  constructor() {
+    this.connectionString = [
+      "mongodb+srv://",
+      data_chunks.DB_user,
+      ":",
+      data_chunks.DB_pass,
+      "@cluster0.br6n1f3.mongodb.net/",
+    ].join("");
+    this.connect();
+  }
+  connect = async () => {
+    await mongoose
+      .connect(this.connectionString)
+      .then(() => {
+        logger.info("CONNECTED");
+      })
+      .catch((err) => {
+        logger.error(err.message);
+      });
+  };
+
+  getData = async (dir) => {
+    return await dir
+      .find()
+      .then((Data) => {
+        return Data;
+      })
+      .catch((err) => {
+        logger.error(err.message);
+      });
+  };
+
+  addData = async (body, dir) => {
```

Comment:
use proper function 
ex : 
function myFunction(p1, p2) {
  return p1 * p2;
}
also specify types for variables

Reply:
done.